operators/creating-a-python-operator.md

```diff
@@ -12,17 +12,15 @@ def my_prediction_function(input: pd.DataFrame) -> pd.DataFrame:
     return predictions
 ```
 
-{% hint style="info" %}
-All Python functions in Aqueduct by default take in Pandas DataFrames and return Pandas DataFrames. We plan to add support for other data types in the future.
-{% endhint %}
-
 When a Python function is annotated with `@op`, two things happen:
 
 1. Aqueduct packages up this code and infers any library dependencies that it might require, and prepares the code to run as a part of a workflow.&#x20;
 2. The annotated function will accept Aqueduct [Artifacts](../artifacts.md) as inputs and return Artifacts as outputs -- the return values can in turn by used by other operators.&#x20;
 
 {% hint style="info" %}
-Note that once an operator has been annotated with `@op`, it cannot be used as a "regular" Python function -- the inputs to the function must be Artifacts.
+Note that once an operator has been annotated with `@op`, it cannot be used as a "regular" Python function.
+The inputs to this operator are expected to be Artifacts. In case you supply an input of another kind, we will implicitly convert the input into the appropiate Artifact representation.
```

Comment:
typo: "appropriate"

Could we slightly tweak this to: "If you supply a non-Artifact input, we will implicitly convert it into a workflow parameter for you, with the default value as the input."